.github/workflows/ci.yml

```diff
@@ -114,6 +114,45 @@ jobs:
         name: android-artifacts
         path: "wrappers/android/zxingcpp/build/outputs/aar/zxingcpp-debug.aar"
 
+  build-kn:
+    runs-on: macos-latest
```

Comment:
I was about to merge this when I realized one final 'issue': the build time for this job is 18minutes. That will annoy me a lot in the future. It seems obvious to me that this is caused by all the cross-compilation for the myriad of targets. And additionally this happens in macos, which is slower than the ubuntu machine in the normal build as well.

Can we just replace this line with `runs-on: ubuntu-latest` and thereby implicitly disabling all the cross compilation? The latter would then only happen via manual workflow dispatch of the publish-kl workflow. This would be analogous to the long running publish-python workflow.

Reply:
done in https://github.com/zxing-cpp/zxing-cpp/pull/719/commits/d3a9d596b352fa643dd24b566f7c294c162d5c25
And now we test for only linuxX64 target